vcap/vcap/backend.py

```diff
@@ -36,16 +36,37 @@ def process_frame(self,
                       options: Dict[str, OPTION_TYPE],
                       state: BaseStreamState) \
             -> DETECTION_NODE_TYPE:
-        """A method that does the preprocessing, inference, and postprocessing
-        work for a frame. It has the ability to call self.send_to_batch(frame)
-        to send work for batching (eventually run in batch_predict)."""
+        """A method that does the pre-processing, inference, and postprocessing
+        work for a frame.
+
+        If the capsule uses an algorithm that benefits from batching,
+        this method may call ``self.send_to_batch``, which will asynchronously
+        send work out for batching. Doing so requires that the
+        ``batch_predict`` method is overridden.
+
+        :param frame: A numpy array representing a frame. It is of shape
+            (height, width, channel) and the frames come in BGR order.
```

Comment:
I don't think `channel` is the perfect term to use here. I would suggest `num_channels`

Reply:
Durn it!